src/test/mochitest/browser_dbg-expressions.js

```diff
@@ -32,7 +32,13 @@ function assertEmptyValue(dbg, index) {
 }
 
 function toggleExpression(dbg, index) {
-  findElement(dbg, "expressionNode", index).click();
+  const node = findElement(dbg, "expressionNode", index);
+  const objectInspector = node.closest(".object-inspector");
+  const properties = objectInspector.querySelectorAll(".node").length;
+  node.click();
+  return waitUntil(
+    () => objectInspector.querySelectorAll(".node").length !== properties
+  );
```

Comment:
shouldn't we wait till it is the same?

can we move this to a helper?

Reply:
nope, since we expand a node, the result should be that the objectInspector has a different number of nodes